src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -176,6 +176,344 @@ namespace cryptonote
   };
 
   //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_GET_ADDRESS_TXS
```

Comment:
This is bikeshedding, but I wouldn't call that LIGHT_WALLET. Other things might have a use for this, so keeping the name factual might be best,